knowledge-base/remove-mergefields-retain-values-radwordsprocessing.md

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
The described scenario throughout the KB suggests that the value of the MergeField is **preserved**. However, the value of the MergeField is actually **replaced** from <<first name>> to "John" and <<last name>> to "Smith".